docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -871,7 +871,7 @@ These paths must follow 1 of 3 syntactic forms:
 
 |Chained Property Path
 |"a.b.c"
-|Refers to a supplier variable reachable via chained properties. Intermediate elements must be facts or variables; the final one must be a supplier variable.
+|Refers to a supplier variable reachable via chained properties. Intermediate elements must be facts or variables; the final one must be a variable.
```

Comment:
What is the difference between a variable and a supplier variable?
The path "refers to a supplier variable", yet "final one must be a variable". It seems like the terms are used interchangeably.